.bazelrc

```diff
@@ -1,7 +1,7 @@
 # TODO(irfansharif): We should fold this into `dev` instead (#56965).
 
-build --ui_event_filters=-DEBUG --define gotags=bazel,crdb_test_off
-test --define gotags=bazel,crdb_test
+build --symlink_prefix=_bazel/ --ui_event_filters=-DEBUG --define gotags=bazel,crdb_test_off
```

Comment:
just get rid of the changes to these two lines and instead add a new line `common --symlink_prefix=_bazel/` :)

Reply:
What is the syntax? The query command doesn't recognize the flag.

Reply:
Ah, no, good catch, you're right. You can keep it the way it is.